mobile/src/main/res/values-zh-rTW/strings.xml

```diff
@@ -120,7 +120,7 @@
     <string name="settings_misc_source_summary">閱讀使用說明, star, 提交 issues, 合作</string>
     <string name="settings_misc_donate">抖內</string>
     <string name="settings_misc_donate_summary">我喜歡錢</string>
-    <string name="settings_misc_donate_more">PayPal, Flattr，等其他方式…</string>
+    <string name="settings_misc_donate_more">PayPal, Flattr,等其他方式...</string>
```

Comment:
Please use the character `…` whenever possible. Also `, ` or `，` is preferred to `,` without space.

Reply:
It should be this one translate.
PayPal， Flattr，等其他方式...

Reply:
Suggested punctuation: `PayPal, Flattr, 等其他方式…`

What do you think?

Reply:
Please tell me which one do you prefer:

1. `PayPal, Flattr, 等其他方式…`
2. `PayPal，Flattr，等其他方式…`

`…` is preferred to `...` to prevent line breaking issues.

Reply:
I like the first one.